rust/lance-index/src/scalar/inverted/query.rs

```diff
@@ -119,6 +120,8 @@ pub enum FtsQuery {
     // compound queries
     Boost(BoostQuery),
     MultiMatch(MultiMatchQuery),
+    // Openapi generator does not support recursive schema
+    #[schema(value_type = Object)]
```

Comment:
@BubbleCal could you help provide some guidance on:
if BoostQuery and BooleanQuery cannot be used for FTS, how much of the FTS query functionality would be affected?